apps/configurator/lib/configurator_web/controllers/utils_api.ex

```diff
@@ -0,0 +1,25 @@
+defmodule ConfiguratorWeb.UtilsAPI do
+  @moduledoc """
+  Utility functions for the Configurator API
+  """
+  alias Configurator.Configuration
+
+  def list_characters do
+    Configuration.list_characters()
+    |> Enum.map(fn character ->
+      %{
+        active: character.active,
+        name: character.name,
+        base_speed: character.base_speed,
+        base_size: character.base_size,
+        base_health: character.base_health,
+        base_stamina: character.base_stamina,
+        max_inventory_size: character.max_inventory_size,
+        natural_healing_interval: character.natural_healing_interval,
+        natural_healing_damage_interval: character.natural_healing_damage_interval,
+        stamina_interval: character.stamina_interval,
+        skills: character.skills
+      }
```

Comment:
What's this for? I'm confused.

Reply:
Oh, the purpose of this is convert the Ecto character struct to map, since `Map.from_struct` doesn't work properly with Ecto structs, there's a couple of hidden keys that get parsed and the json encoder doesn't know how to handle them. 
But I think I can do something that might work